src/main/java/com/photolocos/enterprise/dto/LocationDTO.java

```diff
@@ -1,135 +1,29 @@
 package com.photolocos.enterprise.dto;
+
 import lombok.Data;
 
 import javax.persistence.*;
 import java.io.Serializable;
 import java.util.Date;
 
-
-
 /**
  * @author Max Graman (gramanma@mail.uc.edu)
  * TODO: JavaDoc.
  */
+@SuppressWarnings("ALL")
```

Comment:
I urge caution here.  Turning off warnings incorrectly assumes that you don't know won't hurt you... which could hide a lot of very important information.